embc-app/ClientApp/src/app/provincial-admin/components/volunteer-organization-list/volunteer-organization-list.component.html

```diff
@@ -26,38 +26,36 @@ <h3>Organization Details</h3>
   </section>
 
   <h3 class="blue-header">USERS LIST</h3>
-  <form novalidate role="search" [formGroup]="form">
-    <div class="form-row">
-      <div class="col-md">
-        <input type="text" aria-label="search" class="form-control mb-2" placeholder="Search by last name &hellip;" formControlName="searchbox" (keydown.enter)="search()">
-      </div>
-      <div class="col-md-auto">
-        <button class="btn btn-primary" (click)="search()">SEARCH</button>
-      </div>
+  <div class="form-row">
+    <div class="col-md">
+      <input type="text" aria-label="search" class="form-control mb-2" placeholder="Search by last name &hellip;" [(ngModel)]="queryString" (keydown.enter)="search()">
     </div>
-    <div class="form-row">
-      <div class="form-check form-check-inline">
-        <input type="radio" class="form-check-input" formControlName="userToggle" [value]="SHOW_ALL">
-        <label class="form-check-label">Show all</label>
-      </div>
-      <div class="form-check form-check-inline">
-        <input type="radio" class="form-check-input" formControlName="userToggle" [value]="SHOW_ADMINS_ONLY">
-        <label class="form-check-label">Show only admin users</label>
-      </div>
-      <div class="form-check form-check-inline">
-        <input type="radio" class="form-check-input" formControlName="userToggle" [value]="SHOW_ESS_USERS_ONLY">
-        <label class="form-check-label">Show only Emergency Support Services users</label>
-      </div>
+    <div class="col-md-auto">
+      <button class="btn btn-primary" (click)="search()">SEARCH</button>
     </div>
-  </form>
+  </div>
+  <div class="form-row">
+    <div class="form-check form-check-inline">
+      <input type="radio" class="form-check-input" [(ngModel)]="userType" value="SHOW_ALL">
+      <label class="form-check-label">Show all</label>
```

Comment:
I thought I had done this but I don't see it here... Please add an id to each radio input (eg, showAdminsOnly, etc) and a for="showAdminsOnly" to its label. This will allow a user to click on the label and set the radio button.